inst/REFERENCES.bib

```diff
@@ -2,7 +2,7 @@ @TechReport{TSD_evsynth
   author      = {Dias, S. and Welton, N. J. and Sutton, A. J. and Caldwell, D. M. and Lu, G. and Reken, S. and Ades, A. E.},
   institution = {{National Institute for Health and Care Excellence}},
   title       = {{NICE DSU Technical Support Documents} 1-7: Evidence Synthesis for Decision Making},
-  url         = {https://nicedsu.sites.sheffield.ac.uk},
+  url         = {http://www.nicedsu.org.uk},
```

Comment:
This link has a 503 permanent redirect which is [not allowed](https://cran.r-project.org/web/packages/URL_checks.html) on CRAN. We need to use the target page as the link:

```suggestion
  url         = {https://www.sheffield.ac.uk/nice-dsu},
```

It is unfortunate that we can't just use the nicedsu.org.uk URL - the redirect has changed a few times which keeps breaking things!